apps/wallet/src/manifest/manifest.json

```diff
@@ -6,7 +6,7 @@
 	"background": {
 		"service_worker": "background.js"
 	},
-	"permissions": ["storage", "tabs", "alarms"],
+	"permissions": ["storage", "tabs", "alarms", "unlimitedStorage"],
```

Comment:
we might want to include something in our release notes to inform users why this was added